projects/dotcms-field-elements/src/utils/props/validators/props.spec.ts

```diff
@@ -0,0 +1,145 @@
+import { PropValidationInfo } from '../models';
+import {
+    dateRangeValidator,
+    dateTimeValidator,
+    dateValidator,
+    numberValidator,
+    regexValidator,
+    stringValidator,
+    timeValidator
+} from './props';
+
+describe('Props Validators', () => {
+    let propInfo: PropValidationInfo<any>;
+
+    beforeEach(async () => {
+        propInfo = {
+            field: { type: 'test-type', name: 'field-name' },
+            name: 'test-name',
+            value: 'test-value'
+        };
+    });
+
+    describe('stringValidator', () => {
+        it('should not trow throw Warning exception when value is a string', () => {
+            expect(() => stringValidator(propInfo)).not.toThrowError();
```

Comment:
Why `() =>` instead of just run the function?

Reply:
the `toThrowError` receive a function, do it directly throw an error.